gatsby-browser.js

```diff
@@ -33,3 +33,15 @@ export const wrapRootElement = ({ element }) => (
 )
 
 export const wrapPageElement = ({ element }) => <Layout>{element}</Layout>
+
+export const onInitialClientRender = () => {
```

Comment:
We can move this to the global scope I guess

Reply:
Done